src/main/flight/pid.c

```diff
@@ -629,8 +673,10 @@ static void NOINLINE pidApplyFixedWingRateController(pidState_t *pidState, fligh
 {
     const float rateError = pidState->rateTarget - pidState->gyroRate;
     const float newPTerm = pTermProcess(pidState, rateError, dT);
+    const float newDTerm = dTermProcess(pidState, dT);
     const float newFFTerm = pidState->rateTarget * pidState->kFF;
 
+    DEBUG_SET(DEBUG_FW_D, axis, newDTerm);
```

Comment:
@avsaase we already have D logged in PID BB. It should be rather `DEBUG_FW_FF` which currently is not logged at all

Reply:
I mostly did this so I could have the D-term on the OSD for planes that don't have a blackbox. At some point the blackbix would need to be changed to log both D and FF but for now I chose to log D instead of FF because I thought it would be more informative.